### ISSUE TITLE
undefined is cast as a String

### ISSUE THREAD

**Issue by Nautigsam** [nwoltman/string-natural-compare#5]
It looks like when given `undefined`, the function behaves as if it was the string `'undefined'`.

```javascript
nc = require('string-natural-compare')
nc('a', undefined) // -20
nc('v', undefined) // 1
```

I think this is very error-prone, the function should throw in that case.

**Comment by nwoltman** [comment]
I agree with this. The function should throw if either of the arguments passed to it are not a string.

**Comment by nwoltman** [comment]
I'll include a fix for this in the next release.

**Comment by nwoltman** [comment]
Fixed by 65d397b and released in `v3.0.0`.
